### DISCUSSION TITLE
Show more recent articles on homepage

### DISCUSSION THREAD

**Discussion by gcailly** [jpanther/congo#401] [Ideas]
Hi,
I would like to know of it's possible to change the number of recent articles. It's set to 5 and I cannot find how to increase it to 15 for example. I hope it's not an obvious setting it didn't find... Thank you for your help.

**Comment by maijau** [comment]
Check the theme file layouts/partials/recent-articles.html. For a quick and dirty solution you can override the value there. For a better solution (for which I do not have the time at the moment), you could provide your own partial which reads a parameter and uses that here.

**Comment by gcailly** [comment]
I found the variable in layouts/partials/recent-articles.html in the repository !

```
{{ if .Site.Params.homepage.showRecent | default false }}
  <h2 class="mt-8 text-2xl font-extrabold">{{ i18n "shortcode.recent_articles" | emojify }}</h2>
  {{ range first 5 (.Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 5).Pages }}
    {{ partial "article-link.html" . }}
  {{ end }}
{{ end }}
```

I believe I should replace 5 by 10 or 15 in this line :

```
  {{ range first 5 (.Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 5).Pages }}
```

But I use Congo as a module : I don't know where to modify this file in my local folder because it's empty.

I don't know well how Hugo works and Go (if it's Go) is unknown to me. I don't know how to create a partial. I will see the documentation.

EDIT : I got it. I created /layout/partials/recent-articles.html and copied these lines :

```
{{ if .Site.Params.homepage.showRecent | default false }}
  <h2 class="mt-8 text-2xl font-extrabold">{{ i18n "shortcode.recent_articles" | emojify }}</h2>
  {{ range first 15 (.Paginate (where .Site.RegularPages "Type" "in" .Site.Params.mainSections) 15).Pages }}
    {{ partial "article-link.html" . }}
  {{ end }}
{{ end }}
```

Now I have the 15 last articles and I am happy, thank you.
If you would have the time to implement it properly, it would be great.

**Comment by gcailly** [comment]
Thank you maijau. With your help I searched how to do this. I made a PR : https://github.com/jpanther/congo/pull/411
